src/Microsoft.ComponentDetection.Detectors/yarn/Contracts/YarnBerryLockfile.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Microsoft.ComponentDetection.Detectors.Yarn.Contracts;
+
+using System.Collections.Generic;
+using YamlDotNet.Serialization;
+
+/// <summary>
+/// Represents the yarn.lock file for yarn v2+.
+/// </summary>
+public sealed record YarnBerryLockfile
```

Comment:
Can we add a link to the spec for the yarn lockfile?

Reply:
There is no spec :(